docs/source/quick_start/training_on_reading_MC_tables_in_dlc.md

```diff
@@ -0,0 +1,126 @@
+# 使用TorchEasyRec读取MaxCompute表进行模型训练
+
+此文档提供在阿里云的PAI-DLC（分布式训练容器）中使用TorchEasyRec读取MaxCompute（MC）表进行模型训练的步骤和配置示例。
+
+## 环境准备
+
+### 创建数据集
+1. 登录[PAI控制台](https://pai.console.aliyun.com/)。
+2. 点击 **AI资源管理 - 数据集 -> 创建数据集**。
+3. 选择数据存储为 **阿里云文件存储(NAS)**，并挂载到 `/mnt/data` 下。
+
+### TorchEasyRec安装
```

Comment:
delete this section，should install tzrec in each dlc command, ref dlc_tutorial.md